cibyl/models/openstack/node.py

```diff
@@ -27,4 +28,45 @@ class Node(Model):
     """
     # To be implemented in future PR
 
-    API = {}
+    API = {
+        'name': {
+            'attr_type': str,
+            'arguments': [Argument(name='--name', arg_type=str,
+                          description="Node name")]
+        },
+        'node_id': {
+            'attr_type': str,
+            'arguments': [Argument(name='--id', arg_type=str,
+                          description="Id provided for the node")]
+        },
+        'status': {
```

Comment:
I'm not sure if we really have a use case for status.
If we do, I would make only two options: active and inactive and set active as the default one.